app/utils/interfaces.ts

```diff
@@ -27,3 +27,23 @@ export interface UpdateEvent {
   start: stringOrDate;
   end: stringOrDate;
 }
+
+export interface EventState {
+  events: CalEvent[];
+  addEvent: (event: CalEvent) => void;
+  removeEvent: (event: CalEvent) => void;
+}
+
+export interface User {
+  firstname: string;
+  lastname: string;
+  bio: string;
+  username: string;
+  timezone: string;
+  onboarding: boolean;
```

Comment:
Nitpick:
As this is a boolean can we rename it to  `onboardingComplete` or `isOnboarded`?